src/game.py

```
@@ -1,31 +1,53 @@
+import random
+
+from database.db_manager import DatabaseManager
 from player import Player
 from airport import Airport
+from src.Helpers import get_random_airport
```

Ruff (0.8.2)

`src.Helpers.get_random_airport` imported but unused

Remove unused import: `src.Helpers.get_random_airport`

(F401)